tests/testthat/test-as_latex.R

```diff
@@ -32,7 +32,37 @@ test_that("Table width correctly output in LaTeX", {
 
 })
 
-test_that("Table styles correctly applied", {
+test_that("Table width correctly output in LaTeX using tabular*", {
+
+  gt_latex_width_1 <-
+    gt(exibble) %>%
+    tab_options(table.width = pct(90), latex.use.longtable = FALSE) %>%
+    as_latex()
+
+  expect_match(gt_latex_width_1, "\\\\begin\\{tabular\\*\\}\\{0.9\\\\linewidth\\}\\{@\\{\\\\extracolsep\\{\\\\fill\\}\\}")
+
+  # Test specification of a table width in pixels
+  gt_latex_width_2 <-
+    gt(exibble) %>%
+    tab_options(table.width = '600px', latex.use.longtable = FALSE) %>%
```

Comment:
```suggestion
    tab_options(table.width = "600px", latex.use.longtable = FALSE) %>%
```

Nitpick: We use double quotes in the codebase!